### PR TITLE
refactor: move instance-views into hub

### PR DESCRIPTION
**Pull request by cabreraalex**
# Description

Instead of having a separate instance-views repo, moves it into zeno-hub and creates a playground endpoint.

### CODE REVIEW

File: frontend/src/routes/playground/+layout.ts [hunk 1]
```diff
@@ -0,0 +1 @@
+export const prerender = true;
```
Comment (Sparkier) [inline frontend/src/routes/playground/+layout.ts:1]: I don't think we need this now that we're not using GitHub pages anymore.

**Review by Sparkier — COMMENTED** [review]
I assume this is just copying over stuff, right? No in depth review needed? 

There seem to be 2 package-lock files.

File: frontend/src/routes/playground/+layout.ts (hunk 1 above)
Comment (cabreraalex) [reply frontend/src/routes/playground/+layout.ts:1]: Are you sure? I think sveltekit will still pre-render this if it can and it'll be faster

**Comment by cabreraalex** [comment]
> I assume this is just copying over stuff, right? No in depth review needed?
> 
> There seem to be 2 package-lock files.

Yup didn't change much except the paths and imports, all seems to be working. I'll check the lockfiles.

File: frontend/src/routes/playground/+layout.ts (hunk 1 above)
Comment (Sparkier) [reply frontend/src/routes/playground/+layout.ts:1]: Can definitely leave it if we think it's better just done need it anymore I think.

**Review by Sparkier — APPROVED** [review]

**Comment by Sparkier** [comment]
Will need to update FAQ and zenoml.com

**Comment by cabreraalex** [comment]
[ZEN-334 : update zenoml.com and FAQ with new instance view links](https://linear.app/zenoml/issue/ZEN-334/update-zenomlcom-and-faq-with-new-instance-view-links)

**Pull request merged**